src/components/TextFieldChip/TextFieldChips.tsx

```diff
@@ -165,7 +142,7 @@ const TextFieldChips = React.forwardRef(
     const updateChip = (
       chipValue: MuiChipsInputChip,
       chipIndex: number,
-      event: React.KeyboardEvent<HTMLInputElement>
+      event?: React.KeyboardEvent<HTMLInputElement>
```

Comment:
why it becames optional ?

Reply:
handleClickAway doesn't have an event - line 183-185 I am updating the chips from within that function

Is there a different way you'd prefer for it to be handled?

Reply:
Ahhh ok got it ! No it's good ;) ty !